src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Android.cs

```diff
@@ -97,7 +99,7 @@ private void InitializeScrollbars()
 			}
 		}
 
-		IReadOnlyList<View> IShadowChildrenProvider.ChildrenShadow => Content != null ? new[] { Content } : new View[0];
+		List<View> IShadowChildrenProvider.ChildrenShadow => Content != null ? new List<View>(1) { Content } : _emptyList;
```

Comment:
Ohhh I thought this was optimized by the compiler.

Reply:
It cannot, the variable out of the interface is an interface, which has to go through virtcall for getting the enumerable, then the enumerator.